cypress-tests/cypress/e2e/admin/pageBuilder/templates/templatePreview.cy.ts

```diff
@@ -0,0 +1,73 @@
+import { customAlphabet } from "nanoid";
+
+context("Page Builder - Templates", () => {
```

Comment:
Please check the "`context` And `it`" section of our handbook. We'll need to update the titles in `context`.